server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -1090,6 +1090,40 @@
           "to": []
         }
       }
+    },
+    {
+      "allow": {
+        "from":[
+          {
+            "kind": "Job",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                ["settings", "spec"]
+              ],
+              "description": "A Job in Kubernetes is a resource used to run pods to completion"
```

Comment:
Good descriptions in that both offer a definition of the resource. We're in need of a second sentence in each description that describes how one modifies or is modifies by the next.

Reply:
Currently we don't have a relationship_template.yaml. I'm looking into this!